setup.md

```diff
@@ -203,18 +216,50 @@ $ conda activate molssi_best_practices
 Install the general cookiecutter with the following commands.
 
 ~~~
-$ conda config --add channels conda-forge
-$ conda install cookiecutter
+$ conda install -c conda-forge cookiecutter
 ~~~
 {: .language-bash}
 
-## Installing and configuring Git <a name="git_configuration"></a>
+
+## Installing and configuring git
 
 ### Installation
-Install Git. You can install using conda
+We will be using the `git` software for version control during this workshop. 
+This portion walks you through installing and configuring `git`.
+
+If you do not have the environment activated, activate it first:
+
+~~~
+conda activate msse-python
+~~~
+{: .language-bash}
+
+Next, make sure you have git installed.
+
+You can check if git is installed using the following command in your terminal:
 
 ~~~
-$ conda install -c conda-forge git
+git --version
+~~~
+{: .bash}
+
+Make sure that this outputs at least version 2.28. 
+If you do not have git installed, or if it is an older version of git, 
+you can install git using conda:
+
+~~~
+conda install -c conda-forge git
+~~~
+{: .language-bash}
+
+Note that because of the solver that conda uses to decide which version of a package to install you may end up with a version that is < 2.28. 
+you can use the same command from above `git --version` to see what version has been installed.
+
+If the output of that command is < 2.28 you will want to use the following command to specify the version to install. 
+Any version >=2.28 is acceptable.
+
+~~~
+conda install -c conda-forge git"=>2.28"
```

Comment:
This syntax and quoting are slightly odd, and possibly not what is intended. This equates to git `version='>2.28'`, though I think what is intended is `version='>=2.28'`.

I think what was intended was
```suggestion
conda install -c conda-forge git">=2.28"
```

I guess it is an interesting quirk of the package specification syntax that `conda install git=">=2.28"` is invalid.